src/commands.rs

```diff
@@ -2397,11 +2401,27 @@ fn cmd_resolve(
         commit.id().hex()
     ));
     let new_tree_id = workspace_command.run_mergetool(ui, &commit.tree(), repo_path)?;
-    tx.mut_repo()
+    let new_commit = tx
+        .mut_repo()
         .rewrite_commit(ui.settings(), &commit)
         .set_tree(new_tree_id)
         .write()?;
-    workspace_command.finish_transaction(ui, tx)
+    workspace_command.finish_transaction(ui, tx)?;
+
+    if !args.quiet {
+        let new_tree = new_commit.tree();
+        let new_conflicts = new_tree.conflicts_matching(&EverythingMatcher);
+        if !new_conflicts.is_empty() {
+            ui.write("After this operation, some files at this revision still have conflicts:\n")?;
+            resolve_list(
```

Comment:
Since we just walked the tree to calculate `new_conflicts`, it seem unfortunate to walk it again inside `resolve_list()`. I think we should make that function just print the paths instead, replacing the `tree` and `matcher` arguments by a slice of conflicts instead.

Reply:
That's what I did originally. However, I had to switch to passing the tree in https://github.com/martinvonz/jj/pull/977.  So, I imported the change here. I can undo it here and redo it there again.

Reply:
Specifically, it's in https://github.com/martinvonz/jj/pull/977/commits/0e02b12454564737f776084dba3fa3a6feecf8de.

It does feel a bit wasteful, but there doesn't seem to be any real harm. Let me know if/how you'd like to neaten it up, depending also on what you think of the other PR.

Reply:
Can't still pass in `conflicts` after that commit? I see why you need the `tree` object, but it doesn't seem that `conflicts_matching(&EverythingMatcher)` needs to called again.

Reply:
Sure, that does seem better.